Documentation/criu.txt

```diff
@@ -457,6 +457,8 @@ The 'mode' may be one of the following:
 
     *nftables*::: Use nftables rules to drop the packets.
 
+    *none*::: Don't lock the network. (Useful with *--tcp-close*.)
```

Comment:
We have to mention that CRIU can't dump tcp connections if the network isn't locked. If the `none` mode is used without `--tcp-close`, the user is responsible for locking the network.

Reply:
Done.